src/core/util/sys_vars.cpp

```diff
@@ -1988,7 +1934,8 @@ void mce_sys_var::get_env_params()
         multilock = (multilock_t)temp;
     }
 
-    std::vector<const char *> deprecated_params = {SYS_VAR_TX_NUM_BUFS, SYS_VAR_RX_NUM_BUFS};
+    // Put deprecated parameters here.
+    std::vector<const char *> deprecated_params = {};
```

Comment:
Do we want to keep this code block? It's fast to add it once needed, so maybe let's remove it?

Reply:
Done